libs/elements/layout/src/lib/pages/layout-page/layout-page.component.html

```diff
@@ -0,0 +1,5 @@
+
+
+
+<elewa-group-elewa-hero></elewa-group-elewa-hero>
+
```

Comment:
We don't need a layout page yet. The issue was to just create a reusable hero. The layout should be done in app-component.html